Chap_Terms.tex

```diff
@@ -105,9 +96,9 @@ \section{Naming Conventions}
 \item The \code{pmix_} prefix is used to denote function pointer and type definitions.
 \end{itemize}
 
-Users should not use the \textbf{\code{PMIX}}, \textbf{\code{PMIx}}, or \textbf{\code{pmix}} prefixes in their applications or libraries so as to avoid symbol conflicts with current and later versions of the \ac{PMIx} standard and implementations such as the \ac{PRI}.
-
-%%%%%%%%%%%
+Users shall not use the \textbf{\code{PMIX}}, \textbf{\code{PMIx}}, or \textbf{\code{pmix}} prefixes in their applications or libraries so as to avoid symbol conflicts with \ac{PMIx} implementations.
```

Comment:
I would suggest to add an "_" after the reserved name prefixes, as this allows extensions like "PMIxX_", similar to what is done in MPI

Reply:
Good point

Reply:
No, it isn't a good point. We chose the prefixing rule years ago, and people have built around it. We cannot change it now. The reservation applies to anything starting with "pmix".

Reply:
Per discussion in ASC meeting (7/22/2020):
 * Here we are talking about symbol names, so we should add the `_` after the pmix designators. Allows for `PMIXY`
 * We might add a line here that the "pmix" string prefix is also reserved by PMIx for keys and attributes. Note that "pmixy" is not allowed, but "pmi-xy" would be ok.

Suggested:
> Users shall not have symbols prefixed with ... `PMIX_` (add underscores)
> Users shall not have attributes prefixed with "pmix"

Reply:
Yes, sorry that didn't come across via the github editor. I was not suggesting any name change, but rather to include the underscore in the reservation part, as I assumed that all names have a following underscore anyway.

Reply:
Unfortunately, working thru the document again for v4, this change simply isn't possible as it will break backward compatibility. We cannot include the underscore in the reservation. If people want to extend the definitions, they will need to either prefix "PMIX" or use some completely different identifier.